src/NServiceBus.NHibernate.Tests/TimeoutPersister/When_checking_db_indexes.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace NServiceBus.NHibernate.Tests.TimeoutPersister
+{
+    using System.Data.SqlClient;
+    using System.Threading.Tasks;
+    using global::NHibernate.Cfg;
+    using global::NHibernate.Dialect;
+    using global::NHibernate.Mapping.ByCode;
+    using global::NHibernate.Tool.hbm2ddl;
+    using NUnit.Framework;
+    using TimeoutPersisters.NHibernate.Config;
+    using TimeoutPersisters.NHibernate.Installer;
+
+    class When_checking_db_indexes
+    {
+        SchemaExport schemaExport;
+
+        string dbSchemaName;
```

Comment:
In this case, the reason why those are global variables is that we need them in `TearDown` method to clean-up. Otherwise, the clean-up would have to be duplicated in each method.